components/blocks/videoEmbed.tsx

```diff
@@ -27,6 +28,12 @@ export const VideoEmbed = ({ data }: VideoEmbedProps) => {
       )}
     >
       <VideoModal url={data.url} overflow={data.overflow} />
+      {data.caption && (
+        <p className={classNames("font-bold", !uncentre && "text-centre")}>
+          {data.caption}
```

Comment:
Given that the rule https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions says that the figure should always start with `Video:`

should we hardcode this and strip it from the incoming caption?

Reply:
Yeah, good idea. Do you think maybe there should be a separate string field for duration as well?

Reply:
yep, that should prompt people to put in the duration then